src/module_wrap.cc

```diff
@@ -42,6 +42,101 @@ using v8::Value;
 
 static const char* const EXTENSIONS[] = {".mjs", ".js", ".json", ".node"};
 
+namespace {
+
+Maybe<int64_t> GetTimeoutArg(Environment* env, Local<Value> options) {
+  if (options->IsUndefined() || options->IsString()) {
+    return Just<int64_t>(-1);
+  }
+  if (!options->IsObject()) {
+    env->ThrowTypeError("options must be an object");
+    return Nothing<int64_t>();
+  }
+
+  MaybeLocal<Value> maybe_value =
+      options.As<Object>()->Get(env->context(), env->timeout_string());
+  if (maybe_value.IsEmpty())
+    return Nothing<int64_t>();
+
+  Local<Value> value = maybe_value.ToLocalChecked();
+  if (value->IsUndefined()) {
+    return Just<int64_t>(-1);
+  }
+
+  Maybe<int64_t> timeout = value->IntegerValue(env->context());
+
+  if (timeout.IsJust() && timeout.ToChecked() <= 0) {
+    env->ThrowRangeError("timeout must be a positive number");
+    return Nothing<int64_t>();
+  }
+
+  return timeout;
+}
+
+Maybe<bool> GetBreakOnSigintArg(Environment* env,
+                                Local<Value> options) {
+  if (options->IsUndefined() || options->IsString()) {
+    return Just(false);
+  }
+  if (!options->IsObject()) {
+    env->ThrowTypeError("options must be an object");
+    return Nothing<bool>();
+  }
+
+  Local<String> key = FIXED_ONE_BYTE_STRING(env->isolate(), "breakOnSigint");
+  MaybeLocal<Value> maybe_value =
+      options.As<Object>()->Get(env->context(), key);
+  if (maybe_value.IsEmpty())
+    return Nothing<bool>();
+
+  Local<Value> value = maybe_value.ToLocalChecked();
+  return Just(value->IsTrue());
+}
+
+MaybeLocal<Integer> GetLineOffsetArg(Environment* env,
+                                     Local<Value> options) {
+  Local<Integer> defaultLineOffset = Integer::New(env->isolate(), 0);
```

Comment:
I just moved those functions from node_contextify.cc without modifying them. I'd prefer to leave them like that because the goal is to completely remove them in follow-up PR.

Reply:
I reverted that move. It isn't necessary.